README.md

```diff
@@ -97,229 +93,32 @@ $ apt-get update
 $ apt-get install -y --no-install-recommends postgresql-14
 ```
 
-### Fedora, CentOS, or Red Hat:
-
-```bash
-# Add the repository to your system
-curl https://install.citusdata.com/community/rpm.sh | sudo bash
-
-# Install pg_auto_failover
-sudo yum install -y pg-auto-failover10_11
-
-# Confirm installation
-/usr/pgsql-11/bin/pg_autoctl --version
-```
+### Other installation methods
 
-## Building pg_auto_failover from source
```

Comment:
@DimCitus Was there a reason this section was removed? It's very helpful to someone who wants to use pg_auto_failover on an Amazon Linux system.